.travis.yml

```diff
@@ -0,0 +1,22 @@
+language: cpp
+
+matrix:
+  include:
+# Curl fails on x86_64 travis VMs, someone who knows what they are doing should fix
+#    - os: linux
+#      compiler: gcc
+#      script: cd main/source/linux && make
+#      env: OPJ_CI_ARCH=i386
+#      addons:
+#        apt:
+#          packages:
+#            - gcc-multilib
```

Comment:
I think you need to use gcc-multilib:i386 and same for rest of the packages. Also build-essential.